Onboarding: refer to troubleshooting section (#33961)
home-assistant/home-assistant.github.io: source/getting-started/onboarding.markdown

description: "Instructions to get Home Assistant configured."
---

> After Home Assistant has been [installed](/installation/) on your device, there are 5 steps to log in and see your devices.

Reviewer: "device" twice is somehow confusing. How about:

```suggestion
After Home Assistant has been [installed](/installation/) on your device, there are 5 steps to complete setting up Home Assistant.
```

> 1. Enter the Home Assistant device's address in your browser's address bar, [http://homeassistant.local:8123/](http://homeassistant.local:8123/).

Reviewer: Would be nice if that fits on a single line typically. How about:

```suggestion
1. Enter the following URL into the browser's address bar: [http://homeassistant.local:8123/](http://homeassistant.local:8123/).
```

   - **Option 2: restore from backup**: If you want to restore a backup of a previous installation, select **Restore from backup**.
     - Ignore the rest of this procedure and continue with the procedure on [restoring from backup](/common-tasks/os/#restoring-a-backup).

3. Enter the location of your home.
   - The location is used to configure the time zone, unit system, and currency.
   - It is also used for automations such as showing the weather, opening the shades at sunrise, or starting the vacuum when you leave the home.
   - You can always change this information later in the settings.
   - After finding your location, select **Next**.

    ![Define your location.](/images/getting-started/onboarding_location.png)

4. Select which information you are willing to share.
    - Sharing is disabled by default. However, we would like to encourage you to share some of this data.
    - This information helps us find out which platforms we need to support and where to focus our efforts.
    - The data is anonymized and aggregated. To see the charts we generate out of this data, take a look at our [analytics page](https://analytics.home-assistant.io/).

>     - To confirm, select **Next**.

Author: there is already a comma there? why did you say it needs one?
